inc/environment_indicator/namespace.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace Altis\Cloud\Environment_Indicator;
+
+use function Altis\get_environment_name;
+use function Altis\get_environment_type;
+
+function bootstrap() {
+	add_action( 'admin_bar_init', __NAMESPACE__ . '\\enqueue_admin_scripts' );
+	add_action( 'admin_bar_menu', __NAMESPACE__ . '\\add_admin_bar_env_info' );
+}
+
+/**
+ * Add menu item to admin bar to display environment info:
+ * - environment type
+ * - environment name
+ * - URL to Altis dashboard for the environment
+ */
+function add_admin_bar_env_info() {
+	if ( ! is_admin_bar_showing() ) {
+		return;
+	}
+
+	// Specify capabilities for which environment indicator is shown in the admin bar.
+	$capability = apply_filters( 'altis_env_indicator_capability', 'manage_options' );
+	if ( ! current_user_can( $capability ) ) {
+		return;
+	}
+
+	global $wp_admin_bar;
+	// Add environment menu item to the admin bar.
+	$wp_admin_bar->add_menu( [
+		'id'   => 'altis-env-indicator',
+		'meta' => [
+			'class' => 'altis-env-indicator--' . get_environment_type(),
+		],
+	] );
+
+	// Environment type sub-menu item.
+	$env_type_text = strtoupper( get_environment_type() );
+	$env_type_text = apply_filters( 'altis_env_indicator_text', $env_type_text, get_environment_type() );
+
+	$wp_admin_bar->add_menu( [
+		'id'     => 'altis-env-stack-type',
+		'title'  => $env_type_text,
+		'parent' => 'altis-env-indicator',
+	] );
+
+	// Display environment info for non-local stacks.
+	if ( defined( 'HM_ENV_REGION' ) && get_environment_type() !== 'local' ) {
+		// Environment name sub-menu item.
+		$wp_admin_bar->add_menu( [
+			'id'     => 'altis-env-stack-name',
+			'title'  => esc_html( get_environment_name() ),
+			'parent' => 'altis-env-indicator',
+		] );
+
+		// Environment's Altis dashboard URL sub-menu item.
+		$wp_admin_bar->add_menu( [
+			'id'     => 'altis-env-stack-url',
+			'title'  => __( 'Open in Altis Dashboard', 'altis' ),
```

Comment:
Let's add the external link icon to this too: https://developer.wordpress.org/resource/dashicons/#external

Reply:
I've done it via CSS but somehow I'm seeing a empty square instead of the icon. Not sure what I'm missing. Interestingly enough.. if I use `:before` then it shows ok, but not with `:after` 🤔

Reply:
@dashaluna maybe need to set the font family to dashicons. Let me check

Reply:
oh, use an empty span after the text for the icon instead so `:before` can be used